### ISSUE TITLE
Segfault in Phalcon\Image\Adapter\Imagick

### ISSUE THREAD

**Issue by garyblankenship** [phalcon/cphalcon#10720]
Using Imagick creates a segfault in Phalcon when using the text() method.

```php
$image = new Phalcon\Image\Adapter\Imagick($image_original);
$image->text('testing', 0, 0, 100, '#FFFFFF', 30, '/usr/share/fonts/truetype/dejavu/DejaVuSerif.ttf');
$image->save($image_destination, 80);
```

Jul 24 01:55:56 xxx kernel: [569845.421472] php5-fpm[25359]: segfault at 15 ip 00007ff921fdc14c sp 00007fffe2eda060 error 4 in phalcon.so[7ff91bda4000+6ebb000]

**Comment by quicktoolbox** [comment]
```#0  0x00007f281b04b266 in zephir_call_function_opt (fci=0x7ffe709b3f30, fci_cache=0x7ffe709b3ec0, info=<optimized out>)
    at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:9669
        param = <optimized out>
        i = 0
        original_return_value = <optimized out>
        calling_symbol_table = <optimized out>
        original_op_array = <optimized out>
        original_opline_ptr = <optimized out>
        current_scope = <optimized out>
        current_called_scope = <optimized out>
        calling_scope = 0x1fc0470
        called_scope = 0x1fc0470
        current_this = <optimized out>
        execute_data = {opline = 0x0, function_state = {function = 0x1fc9ff0, arguments = 0x7f282ae835e0}, op_array = 0x0, object = 0x23785f8, 
          symbol_table = 0x0, prev_execute_data = 0x7f282ae834d0, old_error_reporting = 0x0, nested = 0 '\000', 
          original_return_value = 0x7ffe709b4c48, current_scope = 0x24363f8, current_called_scope = 0x24363f8, current_this = 0x2435e60, 
          fast_ret = 0x2435e60, delayed_exception = 0x0, call_slots = 0x7f282ae83568, call = 0x7f282ae83568}
        fci_cache_local = {initialized = 32 ' ', function_handler = 0x1e, calling_scope = 0x236b610, called_scope = 0x800, object_ptr = 0x410}
        fn_flags = <optimized out>
#1  0x00007f281b06f8dd in zephir_call_user_function (info=<optimized out>, params=<optimized out>, param_count=<optimized out>, 
    cache_slot=<optimized out>, cache_entry=<optimized out>, retval_ptr_ptr=<optimized out>, function_name=<optimized out>, 
    type=<optimized out>, obj_ce=<optimized out>, object_pp=0x57)
    at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:11860
        fcall_key = 0x23787e8 "setgravity"
        fci = {size = 72, function_table = 0x1fc0498, function_name = 0x0, symbol_table = 0x0, retval_ptr_ptr = 0x7ffe709b3e98, param_count = 1, 
          params = 0x7ffe709b3f80, object_ptr = 0x23785f8, no_separation = 1 '\001'}
        fcall_key_hash = 56889247896513881
        static_params_array = {0x7ffe709b41b0, 0x24039e0, 0x7ffe709b41b0, 0x7ffe709b41b8, 0x7ffe709b41c0, 0x1fccde0, 0x1fccde0, 0x2377c28, 
          0x1fccde0, 0x0}
        fcic = {initialized = 1 '\001', function_handler = 0x1fc9ff0, calling_scope = 0x1fc0470, called_scope = 0x1fc0470, 
          object_ptr = 0x23785f8}
        temp_cache_entry = 0x0
#2  zephir_call_class_method_aparams (return_value_ptr=0x0, ce=0x0, type=33333232, object=0x23785f8, 
    method_name=0x57 <error: Cannot access memory at address 0x57>, method_len=0, cache_entry=0x0, cache_slot=0, param_count=1, params=0x23787e8)
    at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:12087
        possible_method = 0x0
---Type <return> to continue, or q <return> to quit---
        rv = 0x0
        status = 1889222296
        info = {type = 2, ce = 0x1fc0470, object_ptr = 0x23785f8, class_name = 0x0, class_length = 1889223056, 
          func_name = 0x7f281b3cd8c5 "setgravity", func_length = 10}
#3  0x00007f281b3066f9 in zim_Phalcon_Image_Adapter_Imagick__text (ht=1889222960, return_value=0x0, return_value_ptr=0x1fc9ff0, 
    this_ptr=0x23777d0, return_value_used=87) at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:70032
        params_ = {0x0}
        _6 = 0x0
        _8 = 0x0
        offsetX = 0
        g = 1889222960
        b = 1
        text_param = 0x2374978
        offsetX_param = 0x2377db8
        offsetY_param = 0x2377ed8
        opacity_param = 0x2377e48
        r_param = 0x2377e18
        g_param = 0x2378488
        b_param = 0x2378510
        size_param = 0x2378598
        fontfile_param = 0x2377cf8
        draw = 0x23785f8
        color = 0x2378628
        pixel = 0x2378658
        gravity = 0x0
        _0 = {value = {lval = 139810232390338, dval = 6.9075432761147111e-310, str = {val = 0x7f281b3cd2c2 "rgb(%d, %d, %d)", len = 15}, 
            ht = 0x7f281b3cd2c2, obj = {handle = 456970946, handlers = 0xf}, ast = 0x7f281b3cd2c2}, refcount__gc = 1, type = 6 '\006', 
          is_ref__gc = 0 '\000'}
        _1 = {value = {lval = 0, dval = 0, str = {val = 0x0, len = 680483840}, ht = 0x0, obj = {handle = 0, 
              handlers = 0x7f28288f5c00 <main_arena>}, ast = 0x0}, refcount__gc = 1, type = 1 '\001', is_ref__gc = 0 '\000'}
        _2 = {value = {lval = 0, dval = 0, str = {val = 0x0, len = 15869696}, ht = 0x0, obj = {handle = 0, 
              handlers = 0xf22700 <executor_globals>}, ast = 0x0}, refcount__gc = 1, type = 1 '\001', is_ref__gc = 0 '\000'}
        _3 = {value = {lval = 0, dval = 0, str = {val = 0x0, len = 37110272}, ht = 0x0, obj = {handle = 0, handlers = 0x2364200}, ast = 0x0}, 
          refcount__gc = 1, type = 1 '\001', is_ref__gc = 0 '\000'}
        _4 = 0x23787b8
        _5 = 0x0
---Type <return> to continue, or q <return> to quit---
        _7 = 0x0
        _11 = 0x0
        _12 = 0x0
        text = 0x2374978
        fontfile = 0x2377cf8
#4  0x00000000006f692a in dtrace_execute_internal (execute_data_ptr=<optimized out>, fci=<optimized out>, return_value_used=<optimized out>)
    at /build/php5-oxmf3S/php5-5.6.4+dfsg/Zend/zend_dtrace.c:97
        lineno = <optimized out>
        filename = <optimized out>
#5  0x00007f28251d3776 in xdebug_execute_internal (current_execute_data=0x7ffe709b4430, fci=0x7ffe709b4610, return_value_used=1)
    at /build/buildd/xdebug-2.2.6/xdebug-2.2.6/xdebug.c:1553
        edata = <optimized out>
        fse = 0x2454790
        cur_opcode = <optimized out>
        function_nr = 2035
        restore_error_handler_situation = 0
        tmp_error_cb = <optimized out>
#6  0x00007f281b04beba in zephir_call_function_opt (fci=0x7ffe709b4610, fci_cache=0x7ffe709b45a0, info=<optimized out>)
    at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:9766
        call_via_handler = 0
        i = <optimized out>
        original_return_value = <optimized out>
        calling_symbol_table = <optimized out>
        original_op_array = <optimized out>
        original_opline_ptr = <optimized out>
        current_scope = 0x21beef0
        current_called_scope = 0x21beef0
        calling_scope = 0x21beef0
        called_scope = 0x21beef0
        current_this = 0x23777d0
        execute_data = {opline = 0x0, function_state = {function = 0x21bff40, arguments = 0x7f282ae835e0}, op_array = 0x0, object = 0x23777d0, 
          symbol_table = 0x0, prev_execute_data = 0x7f282ae834d0, old_error_reporting = 0x0, nested = 0 '\000', 
          original_return_value = 0x7ffe709b4c48, current_scope = 0x24363f8, current_called_scope = 0x24363f8, current_this = 0x2435e60, 
          fast_ret = 0x2435e60, delayed_exception = 0x0, call_slots = 0x7f282ae83568, call = 0x7f282ae83568}
        fci_cache_local = {initialized = 7 '\a', function_handler = 0x7ffe709b4470, calling_scope = 0x63, called_scope = 0x0, object_ptr = 0x5f}
        fn_flags = <optimized out>
---Type <return> to continue, or q <return> to quit---
#7  0x00007f281b06f8dd in zephir_call_user_function (info=<optimized out>, params=<optimized out>, param_count=<optimized out>, 
    cache_slot=<optimized out>, cache_entry=<optimized out>, retval_ptr_ptr=<optimized out>, function_name=<optimized out>, 
    type=<optimized out>, obj_ce=<optimized out>, object_pp=0x57)
    at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:11860
        fcall_key = 0x2430690 "_text"
        fci = {size = 72, function_table = 0x21bef18, function_name = 0x0, symbol_table = 0x0, retval_ptr_ptr = 0x7ffe709b4578, param_count = 9, 
          params = 0x7ffe709b4660, object_ptr = 0x23777d0, no_separation = 1 '\001'}
        fcall_key_hash = 16925331950767396927
        static_params_array = {0x7ffe709b4810, 0x7ffe709b4818, 0x7ffe709b4820, 0x7ffe709b4828, 0x7ffe709b4830, 0x7ffe709b4838, 0x7ffe709b4840, 
          0x7ffe709b4848, 0x7ffe709b4850, 0x7f281b041cf0 <zephir_substr+496>}
        fcic = {initialized = 1 '\001', function_handler = 0x21bff40, calling_scope = 0x21beef0, called_scope = 0x21beef0, 
          object_ptr = 0x23777d0}
        temp_cache_entry = 0x0
#8  zephir_call_class_method_aparams (return_value_ptr=0x0, ce=0x0, type=33333232, object=0x23777d0, 
    method_name=0x57 <error: Cannot access memory at address 0x57>, method_len=0, cache_entry=0x0, cache_slot=0, param_count=9, params=0x2430690)
    at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:12087
        possible_method = 0x0
        rv = 0x23785c8
        status = 1889224056
        info = {type = 2, ce = 0x21beef0, object_ptr = 0x23777d0, class_name = 0x21bf720 "\001tn", class_length = 1, 
          func_name = 0x7f281b3cccf0 "_text", func_length = 5}
#9  0x00007f281b2bf475 in zim_Phalcon_Image_Adapter_text (ht=1889224688, return_value=0x2377de8, return_value_ptr=0x1fc9ff0, this_ptr=0x23777d0, 
    return_value_used=87) at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:67196
        params_ = {0x2374978, 0x2377db8, 0x2377ed8, 0x2377e48, 0x2377e18, 0x2378488, 0x2378510, 0x2378598, 0x2377cf8}
        text_param = 0x2374978
        offsetX_param = 0x0
        offsetY_param = 0x0
        opacity_param = 0x0
        color_param = 0x0
        size_param = 0x0
        fontfile_param = 0x0
        colors = 0x2377f08
        _3 = 0x2377d88
        _4 = {value = {lval = 139810232388786, dval = 6.9075432760380321e-310, str = {val = 0x7f281b3cccb2 "hexdec", len = 6}, 
            ht = 0x7f281b3cccb2, obj = {handle = 456969394, handlers = 0x6}, ast = 0x7f281b3cccb2}, refcount__gc = 1, type = 6 '\006', 
          is_ref__gc = 0 '\000'}
---Type <return> to continue, or q <return> to quit---
        _5 = 0x2377db8
        _6 = 0x2377ed8
        _7 = 0x2374bc8
        _8 = 0x2377e18
        _10 = 0x2378510
        _11 = 0x2377e48
        _12 = 0x2378598
        text = 0x2374978
        color = 0x2377d28
        fontfile = 0x2377cf8
#10 0x00000000006f692a in dtrace_execute_internal (execute_data_ptr=<optimized out>, fci=<optimized out>, return_value_used=<optimized out>)
    at /build/php5-oxmf3S/php5-5.6.4+dfsg/Zend/zend_dtrace.c:97
        lineno = <optimized out>
        filename = <optimized out>
#11 0x00007f28251d3776 in xdebug_execute_internal (current_execute_data=0x7f282ae834d0, fci=0x0, return_value_used=0)
    at /build/buildd/xdebug-2.2.6/xdebug-2.2.6/xdebug.c:1553
        edata = <optimized out>
        fse = 0x23f5560
        cur_opcode = <optimized out>
        function_nr = 2029
        restore_error_handler_situation = 0
        tmp_error_cb = <optimized out>
#12 0x00000000007b49c0 in zend_do_fcall_common_helper_SPEC (execute_data=0x7f282ae834d0)
    at /build/php5-oxmf3S/php5-5.6.4+dfsg/Zend/zend_vm_execute.h:560
        ret = 0x7f282ae834b0
        opline = 0x7f2814c8cd40
        fbc = 0x21c2730
        num_args = 456972495
#13 0x0000000000744a00 in execute_ex (execute_data=0x7f282ae834d0) at /build/php5-oxmf3S/php5-5.6.4+dfsg/Zend/zend_vm_execute.h:363
        ret = 0
        original_in_execution = 1 '\001'
#14 0x00000000006f67c8 in dtrace_execute_ex (execute_data=0x7f282ae834d0) at /build/php5-oxmf3S/php5-5.6.4+dfsg/Zend/zend_dtrace.c:73
        lineno = 32552
        scope = 0x0
        filename = 0x0
        funcname = 0x0
---Type <return> to continue, or q <return> to quit---
        classname = 0x0
#15 0x00007f28251d2cfc in xdebug_execute_ex (execute_data=0x7f282ae834d0) at /build/buildd/xdebug-2.2.6/xdebug-2.2.6/xdebug.c:1439
        op_array = 0x2436f98
        edata = <optimized out>
        dummy = 0x716721 <zend_hash_find+465>
        fse = 0x23ac5d0
        xfse = <optimized out>
        magic_cookie = <optimized out>
        do_return = 0
        function_nr = 1989
        le = <optimized out>
        clear = 0
        return_val = 0x0
#16 0x00000000006f865a in zend_call_function (fci=0x7ffe709b4d70, fci_cache=0x0, fci_cache@entry=0x7ffe709b4d40)
    at /build/php5-oxmf3S/php5-5.6.4+dfsg/Zend/zend_execute_API.c:832
        i = 87
        calling_symbol_table = 0xf22868 <executor_globals+360>
        original_opline_ptr = 0x7f282ae833b0
        current_scope = 0x20ae970
        current_called_scope = 0x21e5970
        execute_data = {opline = 0x0, function_state = {function = 0x2436f98, arguments = 0x7f282ae83488}, op_array = 0x0, object = 0x2435e60, 
          symbol_table = 0xf22868 <executor_globals+360>, prev_execute_data = 0x7ffe709b51d0, old_error_reporting = 0x0, nested = 0 '\000', 
          original_return_value = 0x7ffe709b5900, current_scope = 0x0, current_called_scope = 0x0, current_this = 0x0, fast_ret = 0x2399c50, 
          delayed_exception = 0x0, call_slots = 0x7f282ae83458, call = 0x7f282ae83458}
        fci_cache_local = {initialized = 56 '8', function_handler = 0x23749f8, calling_scope = 0x0, 
          called_scope = 0x70ff9b <zend_is_callable_ex+1291>, object_ptr = 0xf22700 <executor_globals>}
#17 0x00007f281b04353b in zephir_call_user_func_array_noex (return_value=0x2421eb8, handler=<optimized out>, params=0x2422190)
    at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:12228
        retval_ptr = 0x0
        fci = {size = 72, function_table = 0x2436420, function_name = 0x23749f8, symbol_table = 0x0, retval_ptr_ptr = 0x7ffe709b4d30, 
          param_count = 0, params = 0x2375ab0, object_ptr = 0x2435e60, no_separation = 1 '\001'}
        fci_cache = {initialized = 1 '\001', function_handler = 0x2436f98, calling_scope = 0x24363f8, called_scope = 0x24363f8, 
          object_ptr = 0x2435e60}
        is_callable_error = 0x0
        status = 0
#18 0x00007f281b3101f8 in zephir_call_user_func_array (params=<optimized out>, handler=<optimized out>, return_value=<optimized out>)
---Type <return> to continue, or q <return> to quit---
    at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:3137
        status = <optimized out>
#19 zim_Phalcon_Dispatcher_dispatch (ht=<optimized out>, return_value=0x23e2418, return_value_ptr=0x7ffe709b5548, this_ptr=0x2421ee8, 
    return_value_used=<optimized out>) at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:21081
        _23 = 0x23749f8
        _4 = 0x0
        _5 = 0x21ea2d0
        _6 = 0x21ea180
        _7 = 0x0
        _10 = 0x0
        _12 = 0x0
        _13 = 0x0
        _14 = 0x20c0a60
        _15 = 0x0
        _16 = 0x0
        _18 = 0x0
        _19 = 0x0
        _21 = 0x0
        _22 = 0x0
        _24 = 0x0
        _25 = 0x0
        _26 = 0x0
        value = 0x2428740
        handler = 0x2435e60
        dependencyInjector = 0x23d45f8
        namespaceName = 0x20b1dc0
        handlerName = 0x2422220
        actionName = 0x24222a8
        params = 0x2422190
        eventsManager = 0x2421f48
        actionSuffix = 0x20b2280
        handlerClass = 0x2422008
        status = 0x0
        actionMethod = 0x2428710
        e = 0x0
        _1 = 0x2421eb8
---Type <return> to continue, or q <return> to quit---
        _2 = 0x0
        _3 = 0x2435e00
        _8 = <optimized out>
        _9 = 0x7f282aeb6280
        _11 = 0x0
        _17 = 0x23777a0
        _20 = <optimized out>
        numberDispatches = 1
        ZEPHIR_LAST_CALL_STATUS = <optimized out>
        hasService = <optimized out>
        wasFresh = <optimized out>
#20 0x00000000006f692a in dtrace_execute_internal (execute_data_ptr=<optimized out>, fci=<optimized out>, return_value_used=<optimized out>)
    at /build/php5-oxmf3S/php5-5.6.4+dfsg/Zend/zend_dtrace.c:97
        lineno = <optimized out>
        filename = <optimized out>
#21 0x00007f28251d3776 in xdebug_execute_internal (current_execute_data=0x7ffe709b51d0, fci=0x7ffe709b53b0, return_value_used=1)
    at /build/buildd/xdebug-2.2.6/xdebug-2.2.6/xdebug.c:1553
        edata = <optimized out>
        fse = 0x2369c40
        cur_opcode = <optimized out>
        function_nr = 1565
        restore_error_handler_situation = 0
        tmp_error_cb = <optimized out>
#22 0x00007f281b04beba in zephir_call_function_opt (fci=0x7ffe709b53b0, fci_cache=0x7ffe709b5340, info=<optimized out>)
    at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:9766
        call_via_handler = 0
        i = <optimized out>
        original_return_value = <optimized out>
        calling_symbol_table = <optimized out>
        original_op_array = <optimized out>
        original_opline_ptr = <optimized out>
        current_scope = 0x21e5970
        current_called_scope = 0x21d7c70
        calling_scope = 0x21e5970
        called_scope = 0x21e5970
        current_this = 0x23ce698
---Type <return> to continue, or q <return> to quit---
        execute_data = {opline = 0x0, function_state = {function = 0x21e9de0, arguments = 0x7f282ae83480}, op_array = 0x0, object = 0x2421ee8, 
          symbol_table = 0xf22868 <executor_globals+360>, prev_execute_data = 0x7f282ae833b0, old_error_reporting = 0x0, nested = 0 '\000', 
          original_return_value = 0x7ffe709b5900, current_scope = 0x0, current_called_scope = 0x0, current_this = 0x0, fast_ret = 0x2399c50, 
          delayed_exception = 0x0, call_slots = 0x7f282ae83458, call = 0x7f282ae83458}
        fci_cache_local = {initialized = 1 '\001', function_handler = 0x3100000007, calling_scope = 0x0, called_scope = 0x0, 
          object_ptr = 0x6e0000005b}
        fn_flags = <optimized out>
#23 0x00007f281b06f8dd in zephir_call_user_function (info=<optimized out>, params=<optimized out>, param_count=<optimized out>, 
    cache_slot=<optimized out>, cache_entry=<optimized out>, retval_ptr_ptr=<optimized out>, function_name=<optimized out>, 
    type=<optimized out>, obj_ce=<optimized out>, object_pp=0x57, object_pp@entry=0x7f281b3cbc3e)
    at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:11860
        fcall_key = 0x2428918 "dispatch"
        fci = {size = 72, function_table = 0x21e5998, function_name = 0x0, symbol_table = 0x0, retval_ptr_ptr = 0x7ffe709b5548, param_count = 0, 
          params = 0x0, object_ptr = 0x2421ee8, no_separation = 1 '\001'}
        fcall_key_hash = 8187311571776318087
        static_params_array = {0x7ffe709b55a0, 0x7ffe709b55a8, 0x7ffe709b55b0, 0x1, 0xf22940 <executor_globals+576>, 0x236c260, 0x2460ef0, 
          0x7f281b19ab91 <zephir_fetch_property_this_quick+337>, 0x0, 0x0}
        fcic = {initialized = 1 '\001', function_handler = 0x21e9de0, calling_scope = 0x21e5970, called_scope = 0x21e5970, 
          object_ptr = 0x2421ee8}
        temp_cache_entry = 0x0
#24 zephir_call_class_method_aparams (return_value_ptr=0x7ffe709b5548, ce=0x0, type=33333232, type@entry=zephir_fcall_method, object=0x2421ee8, 
    method_name=0x57 <error: Cannot access memory at address 0x57>, method_name@entry=0x7f281b3cbc3e "dispatch", method_len=0, 
    method_len@entry=8, cache_entry=0x0, cache_slot=0, param_count=0, params=0x2428918)
    at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:12087
        possible_method = 0x0
        rv = 0x0
        status = 1889228104
        info = {type = 2, ce = 0x21e5970, object_ptr = 0x2421ee8, 
          class_name = 0xfbefdb00b0040300 <error: Cannot access memory at address 0xfbefdb00b0040300>, class_length = 1889227720, 
          func_name = 0x7f281b3cbc3e "dispatch", func_length = 8}
#25 0x00007f281b26d2aa in zim_Phalcon_Mvc_Application_handle (ht=<optimized out>, return_value=0x23e2fe0, return_value_ptr=0x7f282ae83398, 
    this_ptr=0x23ce698, return_value_used=<optimized out>)
    at /build/php-phalcon-nwCQQ1/php-phalcon-2.0.5/cphalcon/build/64bits/phalcon.zep.c:73226
        params_ = 0x7ffe709b55a0
        _6 = 0x1
        ZEPHIR_LAST_CALL_STATUS = <optimized out>
---Type <return> to continue, or q <return> to quit---
        returnedResponse = <optimized out>
        _3 = <optimized out>
        uri = 0x7f282aeb6220
        dependencyInjector = 0x23d45f8
        eventsManager = 0x23d6a98
        router = 0x23d5118
        dispatcher = 0x2421ee8
        response = 0x0
        view = 0x23e6a48
        module = 0x0
        moduleObject = 0x2421f18
        moduleName = 0x21d9290
        className = 0x0
        path = 0x7f28288f5c00 <main_arena>
        implicitView = 0x21d94a0
        controller = 0x23e2418
        possibleResponse = 0x0
        renderStatus = 0x0
        _1 = 0x23e7ba8
        _2 = 0x2413028
        _4 = 0x0
        _5 = 0x20c7010
        _7 = 0x2422220
        _8 = 0x24222a8
        _9 = 0x2422190
        _10 = 0x0
#26 0x00000000006f692a in dtrace_execute_internal (execute_data_ptr=<optimized out>, fci=<optimized out>, return_value_used=<optimized out>)
    at /build/php5-oxmf3S/php5-5.6.4+dfsg/Zend/zend_dtrace.c:97
        lineno = <optimized out>
        filename = <optimized out>
#27 0x00007f28251d3776 in xdebug_execute_internal (current_execute_data=0x7f282ae833b0, fci=0x0, return_value_used=1)
    at /build/buildd/xdebug-2.2.6/xdebug-2.2.6/xdebug.c:1553
        edata = <optimized out>
        fse = 0x2369fd0
        cur_opcode = <optimized out>
        function_nr = 1459
---Type <return> to continue, or q <return> to quit---
        restore_error_handler_situation = 0
        tmp_error_cb = <optimized out>
#28 0x00000000007b49c0 in zend_do_fcall_common_helper_SPEC (execute_data=0x7f282ae833b0)
    at /build/php5-oxmf3S/php5-5.6.4+dfsg/Zend/zend_vm_execute.h:560
        ret = 0x7f282ae83390
        opline = 0x7f2814a427d0
        fbc = 0x21d8890
        num_args = 456972495
#29 0x0000000000744a00 in execute_ex (execute_data=0x7f282ae833b0) at /build/php5-oxmf3S/php5-5.6.4+dfsg/Zend/zend_vm_execute.h:363
        ret = 0
        original_in_execution = 0 '\000'
#30 0x00000000006f67c8 in dtrace_execute_ex (execute_data=0x7f282ae833b0) at /build/php5-oxmf3S/php5-5.6.4+dfsg/Zend/zend_dtrace.c:73
        lineno = 32766
        scope = 0x0
        filename = 0x0
        funcname = 0x0
        classname = 0x0
#31 0x00007f28251d2cfc in xdebug_execute_ex (execute_data=0x7f282ae833b0) at /build/buildd/xdebug-2.2.6/xdebug-2.2.6/xdebug.c:1439
        op_array = 0x7f282aeb9c48
        edata = <optimized out>
        dummy = 0xf22700 <executor_globals>
        fse = 0x236bc80
        xfse = <optimized out>
        magic_cookie = <optimized out>
        do_return = 0
        function_nr = 0
        le = <optimized out>
        clear = 1
        return_val = 0x0
#32 0x00000000007091c0 in zend_execute_scripts (type=1, type@entry=8, retval=retval@entry=0x0, file_count=33333232, file_count@entry=3)
    at /build/php5-oxmf3S/php5-5.6.4+dfsg/Zend/zend.c:1344
        files = {{gp_offset = 40, fp_offset = 796405605, overflow_arg_area = 0x7ffe709b5a40, reg_save_area = 0x7ffe709b59d0}}
        i = 1
        file_handle = 0x7ffe709b7e90
#33 0x00000000006a4ecb in php_execute_script (primary_file=0x7ffe709b7e90) at /build/php5-oxmf3S/php5-5.6.4+dfsg/main/main.c:2584
        realfile = "\200l\233p\376\177\000\000\310p\233p\376\177\000\000\000'\362\000\000\000\000\000vK\343+\016\033\005\343\330$\350*(\177\000\00---Type <return> to continue, or q <return> to quit---
0\370!\350*(\177\000\000\330$\350*(\177\000\000\000\000\000\000\000\000\000\000vK\343\274\334\373\371\034vK%\264\064\032\005\343\000\000\000\000(\177\000\000\230\321\353*(\177\000\000\001\000\000\000\000\000\000\000\bm\233p\376\177\000\000\320l\357\001\000\000\000\000`l\357\001", '\000' <repeats 12 times>, "pC\353*(\177\000\000\360R\324\001\000\000\000\000`\003\000\000\000\000\000\000\360\002\000\000\000\000\000\000pC\353*(\177\000\000\360R\324\001\000\000\000\063\067\064\067\071\060\061\061\000 \322\v\002\000\000\000\000\000"...
        __orig_bailout = 0x7ffe709b8120
        __bailout = {{__jmpbuf = {140730787660064, 2087976827038681974, 139810495276216, 139810495275512, 139810495276216, 0, 
              2087976828307458934, -2088233994584241290}, __mask_was_saved = 0, __saved_mask = {__val = {16358510035218942838, 0, 
                140730787654784, 140730787654784, 1024, 0, 18153969418953294592, 0, 0, 139810495275512, 139810495276216, 139810495275512, 
                139810495276216, 0, 7527532, 139810495279896}}}}
        prepend_file_p = 0x0
        append_file_p = 0x0
        prepend_file = {type = ZEND_HANDLE_FILENAME, filename = 0x0, opened_path = 0x0, handle = {fd = 0, fp = 0x0, stream = {handle = 0x0, 
              isatty = 0, mmap = {len = 0, pos = 0, map = 0x0, buf = 0x0, old_handle = 0x0, old_closer = 0x0}, reader = 0x0, fsizer = 0x0, 
              closer = 0x0}}, free_filename = 0 '\000'}
        append_file = {type = ZEND_HANDLE_FILENAME, filename = 0x0, opened_path = 0x0, handle = {fd = 0, fp = 0x0, stream = {handle = 0x0, 
              isatty = 0, mmap = {len = 0, pos = 0, map = 0x0, buf = 0x0, old_handle = 0x0, old_closer = 0x0}, reader = 0x0, fsizer = 0x0, 
              closer = 0x0}}, free_filename = 0 '\000'}
        old_cwd = 0x7ffe709b5a40 "/"
        retval = 0
#34 0x00000000004661a7 in main (argc=719856824, argv=0x7f282ae821f8) at /build/php5-oxmf3S/php5-5.6.4+dfsg/sapi/fpm/fpm/fpm_main.c:2043
        __bailout = {{__jmpbuf = {0, 2087976829472426870, 4, 15755588, 4294967295, 0, 2087976827040779126, -2088233823421772938}, 
            __mask_was_saved = 0, __saved_mask = {__val = {0 <repeats 16 times>}}}}
        c = 719856824
        file_handle = {type = ZEND_HANDLE_FILENAME, filename = 0x7f282ae83318 "/home/x/public/index.php", opened_path = 0x0, 
          handle = {fd = 720084208, fp = 0x7f282aeb9cf0, stream = {handle = 0x7f282aeb9cf0, isatty = 0, mmap = {len = 1268, pos = 0, map = 0x0, 
                buf = 0x7f282aee4000 <error: Cannot access memory at address 0x7f282aee4000>, old_handle = 0x0, old_closer = 0x0}, 
              reader = 0x6bbff0 <_php_stream_read>, fsizer = 0x6a10c0 <php_zend_stream_fsizer>, 
              closer = 0x6a10a0 <php_zend_stream_mmap_closer>}}, free_filename = 0 '\000'}
        max_requests = 0
        fcgi_fd = 1889238672
        request = {listen_socket = 0, fd = 5, id = 1, keep = 0, closed = 0, in_len = 0, in_pad = 0, out_hdr = 0x0, 
          out_pos = 0x7ffe709b83c0 "\001\003", 
          out_buf = "\001\003\000\001\000\b\000\000\000\000\000\000\000a: no-cache\r\nStatus: 200 OK\r\nContent-type: text/css;charset=UTF-8\r\nCache-Control: public, max-age=31536000, s-maxage=31536000\r\nExpires: Sun, 24 Jul 2016 15:42:50 GMT\r\n\r\n\000\000\001\006\000\001\377\370\000\000>\n<font "..., reserved = '\000' <repeats 15 times>, env = 0x7f282ae82040}
        __func__ = "main"
```

**Comment by andresgutierrez** [comment]
Fixed in the 2.0.x branch
